pkgs/top-level/all-packages.nix

```diff
@@ -20666,6 +20666,8 @@ in
 
   polymake = callPackage ../applications/science/math/polymake { };
 
+  pop-gtk-theme = callPackage ../data/themes/pop-gtk { };
```

Comment:
This file is already a bit messy, but further down there is `### DATA` section that contains the packages located under pkgs/data, such as themes.
So this line should rather be between `polytopes_db` and `posix_man_pages`